src/routes/AdminRoute.tsx

```diff
@@ -15,6 +16,10 @@ export const AdminRoute: React.FC = observer(() => {
             </p>
             <Button onClick={() => (todosStore.toggle = !todosStore.toggle)}>Toggle {`${todosStore.toggle}`}</Button>
             <p>{JSON.stringify(todosStore.todos)}</p>
+            <Avatar 
+                size="l" 
+                className={`avatar`} 
```

Comment:
Avatar komponenta mora po defaultu met avatar classname. Z classname propertyjim lahko mogoče customizaš kaki padding, margin, etc... design komponente se nastavlja samo v komponenti not, ne pa od zunaj.

Reply:
se pravi, ko kličem komponento "Avatar" ima ona kot property "size", "className" property nima določen ima pa v sami komponenti className "avatar"? če bi pa jaz želel spreminjati class name za razlicne avatarje pa bi to naredil tako, da bi "avatar" komponenti dodal property className?    Recimo tako:export const Avatar: React.FC<IAvatarProps> = observer(({ source, size, onClick, className}) => {
    return (
        <img 
        src={source} 
        onClick={onClick} 
        className={`avatar ${className}`}/>
    )
}).  Ko bi pa komponento klical bi ji dodelil "className" property z neko vrednostjo?